docs/getting-started/steps-to-build-deploy-contract.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Steps to build and deploy a smart contract
```

Comment:
Consider changing to gerund for consistency with the writing guidelines:
'Building and deploying a smart contract'

Reply:
Good point. I'll rename the file as well to "building-deploying-smart-contract.md".